smart-contracts/osmosis/contracts/cl-vault/src/vault/range.rs

```diff
@@ -195,13 +203,47 @@ pub fn handle_initial_create_position_reply(
 
     let unused_pair_balances = get_unused_pair_balances(&deps, &env, &pool_config)?;
 
+    if unused_pair_balances
```

Comment:
This if condition means that we still have to swap some amount of tokens, combined with the middleware, we always swap 100% of the swappable tokens and thus cannot do easy inventory management. I don't think we should fix that for this PR

Reply:
Alright, think I misunderstood then